etna/api/urls.py

```diff
@@ -248,9 +253,111 @@ class CustomImagesAPIViewSet(ImagesAPIViewSet):
     ]
 
 
+class BlogsAPIViewSet(CustomPagesAPIViewSet):
+    filter_backends = []
+    known_query_parameters = []
+
+    def blogs_list_view(self, request):
+        queryset = self.get_queryset()
+        queryset = queryset.not_type(BlogPage)
+        blogs = []
+        for blog in queryset.iterator():
+            blogs_children = blog.get_children().type(BlogPage)
+            blogs_children = DefaultPageSerializer(blogs_children, many=True)
+            if blogs_children.data:
+                blogs += blogs_children.data
+
+        top_level_queryset = self.get_queryset()
+        top_level_queryset = top_level_queryset.type(BlogIndexPage)
+        top_level = DefaultPageSerializer(top_level_queryset, many=True)
+        blogs = top_level.data + sorted(blogs, key=lambda x: x["title"])
+
+        return Response(blogs)
+
+    def authors_view(self, request):
+        # TODO: Get all people who are listed as an author on a blog post
```

Comment:
@jamesbiggs - Is this something that is fairly easy for you to do?

Reply:
Is this so that the author info appears in the blog listing API?

Reply:
It's meant to be a list of all the `PersonPage`s of people who are linked to a `BlogPostPage` as an author.

That way we can get a list of all the authors and display it using the endpoint and filter you created above.

Reply:
To do for later